index.html

```diff
@@ -7,16 +7,23 @@
     <title>John's Portfolio</title>
     <link rel="stylesheet" href="styles.css">
     <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+    <script src="https://kit.fontawesome.com/c6cb015206.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <div class="drop-down-menu hidden-menu"> 
+        <i class="fa-regular fa-x dropdown-x-icon"></i>
+        <ul class="nav-dropdown">
+            <li class="nav-dropdown-item"><a href="#my-work" class="nav-dropdown-item-link" href="#">Portfolio</a></li>
+            <li class="nav-dropdown-item"><a href="#about-group" class="nav-dropdown-item-link" href="#">About</a></li>
+            <li class="nav-dropdown-item"><a href="#contacts" class="nav-dropdown-item-link" href="#">Contact</a></li>
+        </ul>
+    </div>
```

Comment:
- [ ] I would implore you to display this aspect on your desktop version as none so that it would not show it. At the moment, it still shows all. But it works fine on the mobile version, just display: none for the div would suffice.
![image](https://user-images.githubusercontent.com/86200356/217229600-fbeebe75-e470-4341-81f6-d3821765896c.png)